documentation/modules/proc-accessing-kafka-using-routes.adoc

```diff
@@ -36,13 +43,20 @@ spec:
 +
 [source,shell,subs=+quotes]
 oc apply -f _KAFKA-CONFIG-FILE_
++
+`ClusterIP` type services are created for each Kafka broker, as well as an external _bootstrap service_.
+The bootstrap service routes external traffic to the Kafka brokers.
+An OpenShift `Route` resource is also created for each service to expose them using the HAProxy load balancer.
+DNS addresses used for connection are propagated to the `status` of each service.
 
-. Find the address of the bootstrap `Route`.
+. Retrieve the address of the bootstrap service to access the Kafka cluster.
 +
 [source,shell,subs=+quotes]
 oc get routes _KAFKA-CLUSTER-NAME_-kafka-bootstrap -o=jsonpath='{.status.ingress[0].host}{"\n"}'
```

Comment:
Again, you can just use the command `kubectl get kafka my-cluster -o=jsonpath='{.status.listeners[?(@.type=="external")].bootstrapServers}{"\n"}'`